lib/runners/harness.rb

```diff
@@ -91,5 +94,33 @@ def handle_error(exn)
         trace_writer.error "#{exn.message} (#{exn.class})"
       end
     end
+
+    def remove_ignored_files(config)
+      ignores = config.ignore
+      return if ignores.empty?
+
+      trace_writer.message("Deleting specified files via the `ignore` option...") do
```

Comment:
[IMO] This message implicitly depends on `ignore:` option of `sider.yml`, and we may forget to update if the name will change in the future.

So, what about changing the message like this?

```suggestion
      trace_writer.message("Deleting ignored files...") do
```

Reply:
Looks good. I updated with 740f7e7